app/renderer/components/preferences/extensionsTab.js

```diff
@@ -129,13 +129,7 @@ class ExtensionsTab extends ImmutableComponent {
 }
 
 const styles = StyleSheet.create({
-  table: {
-    // TODO (Suguru): refactor sortableTable.js to remove !important
-    width: '100% !important',
-    marginBottom: '0 !important'
-  },
```

Comment:
`table` is no longer required thanks to `fillAvailable`. Also margin is no longer specified by default.